Dockerfile

```diff
@@ -1,33 +1,58 @@
 FROM python:3.6.5-jessie
 
 env DEBIAN_FRONTEND noninteractive
-RUN apt-get update && \
-    apt-get -y install \
+RUN echo 'deb http://deb.debian.org/debian jessie-backports contrib non-free main' >> /etc/apt/sources.list && \
+    apt-get update && \
+    apt-get -y install --no-install-recommends \
+            build-essential \
+            chromedriver \
+            chrpath \
             dbus-x11 \
+            ffmpeg \
             libdbus-1-dev \
+            libdbus-glib-1-dev \
             libespeak1 \
+            libffi-dev \
+            libfontconfig1 \
+            libfontconfig1-dev \
+            libfreetype6 \
+            libfreetype6-dev \
             libnotify-cil-dev \
+            libssl-dev \
+            libxft-dev \
             notification-daemon \
+            python3-dbus \
+            python-dbus \
```

Comment:
Remove this line, it will install python2, I don't think you need it if you are using python3.
 The same goes for `python-notify2` . You can install `python3-notify2` instead. 
`python-imdbpy` will work for python3 you can keep this.